lib/backgrounder/orm/activemodel.rb

```diff
@@ -27,7 +27,12 @@ def _define_shared_backgrounder_methods(mod, column, worker)
         end
 
         def _supported_callback
-          respond_to?(:after_commit) ? :after_commit : :after_save
+          test = Rails.env.test? rescue ENV['RAILS_ENV'] == 'test'
+          if respond_to?(:after_commit) and not test
```

Comment:
Not a fan of this. `after_commit` is pretty standard and there are ways to test it. Changing the functionality for a test is bad practice since you are not actually testing the code that will run in production.